packages/core/src/components/information-bar/information-bar.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, Element, h, Host, Prop, Watch } from '@stencil/core';
+
+export type BarNumbers = {
+  count: number;
+  stripped: number;
+  icon: string;
+  color: string;
+};
+
+@Component({
+  tag: 'information-bar',
```

Comment:
```suggestion
  tag: 'ix-information-bar',
```

If suggestion is applied all test cases has also to be adapted unit und snapshottests.